tools/TZupdate.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+dir=$(cd ${0%/*}; pwd)
+base=${0##*/}
+
+csv=https://raw.githubusercontent.com/nayarsystems/posix_tz_db/master/zones.csv
+
+input=/tmp/.tempfile-zones.csv
+names=/tmp/.tempfile-names.txt
+values=/tmp/.tempfile-values.txt
```

Comment:
`tz_tmpdir=$(mktemp -d)` and `trap 'rm -r $tz_tmpdir` EXIT`?

Reply:
Thanks :+1: It looks better now